Test/TestCase/SearchManagerTest.php

```diff
@@ -153,17 +165,154 @@ public function testRemove()
         $id = $view->getId();
         $searchManager->remove($view);
         $searchManager->flush();
+        sleep(1);
 
         $view = $viewRepo->find($id);
         $this->assertNull($view);
     }
 
+    public function testRemoveTimeSeries()
+    {
+        $this->createTimeSeriesData();
+        $searchManager = $this->getSearchManager();
+        $statusLogRepo = $searchManager->getRepository(StatusLog::class);
+
+        $statusLog = new StatusLog();
+        $statusLog->setId(uniqid());
+        $statusLog->setStatus('failure');
+        $statusLog->setDate(new \DateTime('c'));
+        $statusLogRepo->save($statusLog, true);
+
+        $statusLog = $statusLogRepo->findOneBy(array('status' => 'failure'));
+        $id = $statusLog->getId();
+        $searchManager->remove($statusLog);
+        $searchManager->flush();
+        sleep(1);
+
+        $statusLog = $statusLogRepo->find($id);
+        $this->assertNull($statusLog);
+    }
+
+    public function testRemoveMultiple()
+    {
+        $this->createData();
+        $searchManager = $this->getSearchManager();
+        $viewRepo = $searchManager->getRepository(View::class);
+
+        $viewIds = array('multiple1', 'multiple2', 'multiple3');
+        $views = array();
+        foreach($viewIds as $viewId) {
+            $view = new View();
+            $view->setId($viewId);
+            $view->setBrowser('safari');
+            $view->setDevice('ios');
+            $view->setViews(10);
+            $view->setTags(array(new Tag('pro', 10.0)));
+            $view->setDate(new \DateTime('c'));
+            $viewRepo->save($view, true);
+
+            $view = $viewRepo->findOneBy(array('id' => $viewId));
+            $this->assertNotNull($view);
+
+            $views[] = $view;
+        }
+
+        $searchManager->remove($views);
+        $searchManager->flush();
+        sleep(1);
+
+        foreach($viewIds as $viewId) {
+            $view = $viewRepo->find($viewId);
+            $this->assertNull($view);
+        }
+    }
+
+    public function testRemoveMultipleTimeSeries()
+    {
+        $this->createTimeSeriesData();
+        $searchManager = $this->getSearchManager();
+        $statusLogRepo = $searchManager->getRepository(StatusLog::class);
+
+        $statusLogIds = array('multiple1', 'multiple2', 'multiple3');
+        $statusLogs = array();
+        foreach($statusLogIds as $statusLogId) {
+            $statusLog = new StatusLog();
+            $statusLog->setId($statusLogId);
+            $statusLog->setStatus('failure');
+            $statusLog->setDate(new \DateTime('c'));
```

Comment:
I was expecting this to test the deletion across multiple indices. This creates 3 documents in the same index right?